.pre-commit-config.yaml

```diff
@@ -15,35 +16,50 @@ repos:
       - id: name-tests-test
       - id: check-ast
       - id: debug-statements
+
   - repo: https://github.com/pre-commit/mirrors-mypy
     rev: v1.0.1
     hooks:
       - id: mypy
+
   - repo: https://github.com/asottile/add-trailing-comma
     rev: v2.4.0
     hooks:
       - id: add-trailing-comma
+
   - repo: https://github.com/Lucas-C/pre-commit-hooks
     rev: v1.4.2
     hooks:
       - id: remove-tabs
+
   - repo: https://github.com/pycqa/pydocstyle
     rev: 6.3.0
     hooks:
       - id: pydocstyle
         exclude: ^tests/.*
-  - repo: https://github.com/PyCQA/pylint
-    rev: v2.16.3
+
+  - repo: local
     hooks:
       - id: pylint
-        args:
-          - --jobs=0
-          - --rcfile=.pylintrc
-          - --output-format=colorized
-          - --score=n
-          - --enable=local
-          - --disable=C0330,R0913,R0914
+        name: pylint
+        entry: pylint
+        language: system
+        types: [python]
+        require_serial: true
+
   - repo: https://github.com/PyCQA/flake8
-    rev: 3.9.2
+    rev: 6.0.0
```

Comment:
Changing linter versions is a functional change. Integrating yamllint should be about formatting.